src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs

```diff
@@ -30,7 +42,33 @@ public void ShouldStartIfValidSeedNodesIfSupplied(string seedNodes)
                 var myConfig = ConfigurationFactory.Empty.BootstrapFromDocker();
                 myConfig.HasPath("akka.cluster.seed-nodes").Should().BeTrue();
                 var seeds = myConfig.GetStringList("akka.cluster.seed-nodes").Select(x => x.Trim());
-                seeds.Should().BeEquivalentTo(seedNodes.Split(",").Select(x => x.Trim()));
+                seeds.Should().BeEquivalentTo(seedNodes.Replace("\"", "").Split(",").Select(x => x.Trim()));
```

Comment:
Can you add one more test case that does a hard coded apples to apples comparison of seed nodes quoted different ways, where the expected values are constant and not modified via trim / unquoting in any way? Just want a sanity check in there should anyone ever move some of this stuff around again in the future.

Reply:
Sure, give me a minute

Reply:
Done